ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/project/node/ProjectDescriptorNode.java

```diff
@@ -63,6 +58,13 @@ public void updatePresentation(@NotNull NodePresentation presentation) {
         presentation.setPresentableText(getData().getName());
         presentation.setPresentableIcon(isValid(getData()) ? nodeManager.getNodesResources().projectFolder()
                                                            : nodeManager.getNodesResources().notValidProjectFolder());
+        presentation.setPresentableTextCss("font-weight:bold");
+
+        if ("private".equals(getData().getVisibility())) {
```

Comment:
Do we show some icon If visibility equals to public? If no, why? I think it is not good behaviour to show icon for private project but not show some icon for public projects.